### PR TITLE
Add guard to uploaded_files migration

### PR DESCRIPTION
**Pull request by kirkkwang**
This commit will check if uploaded_files is a table before adding the file_name column to it.  In a Hydra application, it is not so this was crashing.

### CODE REVIEW

File: db/migrate/20240806161142_add_file_name_to_uploaded_files.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class AddFileNameToUploadedFiles < ActiveRecord::Migration[5.2]
   def change
-    add_column :uploaded_files, :filename, :string unless column_exists?(:uploaded_files, :filename)
+    if table_exists?(:uploaded_files)
+      add_column :uploaded_files, :filename, :string unless column_exists?(:uploaded_files, :filename)
+    end
```
Comment (orangewolf) [inline db/migrate/20240806161142_add_file_name_to_uploaded_files.rb:5]:
```suggestion
    add_column :uploaded_files, :filename, :string if table_exists?(:uploaded_files) && !column_exists?(:uploaded_files, :filename)
```

Comment (orangewolf) [reply db/migrate/20240806161142_add_file_name_to_uploaded_files.rb:5]: either this or do both in the `if` block. don't do both `if` and `unless`

**Review by orangewolf — CHANGES_REQUESTED** [review]
see code suggestion

**Comment by kirkkwang** [comment]
Sounds good

**Review by orangewolf — APPROVED** [review]

**Pull request merged**